frontend/src/boot/api.ts

```diff
@@ -8,10 +8,10 @@ declare module '@vue/runtime-core' {
   }
 }
 
-const cdnUrl = 'https://enacit4r-cdn.epfl.ch/';
-
-const baseUrl = `${process.env.API_URL}${process.env.API_PATH}`;
+const appEnv = window.env;
```

Comment:
instead of doing this. Which failed in some cases

I did use a proxy: https://github.com/EPFL-ENAC/unhcr-geneva-tech-hub-app/blob/main/frontend/src/config.ts
here: https://github.com/EPFL-ENAC/unhcr-geneva-tech-hub-app/blob/main/frontend/src/App.vue

Just so you know..